modules/core/src/main/scala/org/scalasteward/core/io/ProcessAlg.scala

```diff
@@ -32,16 +33,20 @@ trait ProcessAlg[F[_]] {
 
 object ProcessAlg {
   abstract class UsingFirejail[F[_]](config: Config) extends ProcessAlg[F] {
-    override def execSandboxed(command: Nel[String], cwd: File): F[List[String]] =
+    override def execSandboxed(command: Nel[String], cwd: File): F[List[String]] = {
       if (config.disableSandbox) {
-        exec(command, cwd)
+        val envVars = config.envVars.map(EnvVar.unapply(_).get)
+        exec(command, cwd, envVars: _*)
       } else {
         val whitelisted = (cwd.pathAsString :: config.whitelistedDirectories)
           .map(dir => s"--whitelist=$dir")
         val readOnly = config.readOnlyDirectories
           .map(dir => s"--read-only=$dir")
-        exec(Nel("firejail", whitelisted ++ readOnly) ::: command, cwd)
+        val envVars = config.envVars
+          .map(envVar => s"--env=${envVar.name}=${envVar.value}")
+        exec(Nel("firejail", whitelisted ++ readOnly ++ envVars) ::: command, cwd)
```

Comment:
@mwz Have you tried passing the env vars as third parameter here instead of usig firejail's `--env`?
```scala
val envVars = config.envVars.map(EnvVar.unapply(_).get)
exec(Nel("firejail", whitelisted ++ readOnly) ::: command, cwd, envVars: _*)
```
If this works, this feature could be independent of firejail.

Reply:
This should work as firejail will inherit any environment variables from the parent process.

I originally thought that when you run firejail it sandboxes a new shell session with any env variables exported in the current shell being discarded (that's why I used the `--env` flag) but apparently, it's not the case. I gave it some thought and in my opinion, the sandboxed execution model could be a bit more strict in such a way, that firejail should not inherit any environment variables from its parent process. I'm not sure how you run scala-steward yourself, but I have the github token (for my scala-steward user) exported as an env variable in the CI and it would make sense to me if it wasn't exposed in the sandbox.

I think this could be achieved by wrapping firejail it in the following command which clears the environment: `env -i bash -lc 'firejail ...'` with the additional environment variables injected explicitly to firejail via the use of `--env`.

I'm happy, for the purpose of this PR, to make the change you suggested, but I would be keen to discuss this a bit further and open another PR with the changes, if you agree with my idea.

Reply:
That's a good point I haven't thought about and I agree that `execSandboxed` should not inherit the environment from scala-steward.

So my current understanding is that environment variables injected via `--env-var` to scala-steward
 - are the only environment variables available to `execSandboxed`
 - and are added to the environment when calling `exec`.

Does that sound good to you?

It is your call if you want to do the changes in this PR or in a follow-up PR. I'm happy either way. :-)

Reply:
> So my current understanding is that environment variables injected via `--env-var` to scala-steward
> 
> * are the only environment variables available to `execSandboxed`
> * and are added to the environment when calling `exec`.
> 
> Does that sound good to you?

Yes, that sounds good 👍 I've made the changes you suggested before, so this PR should be done. I'll follow up with another PR later to implement the rest.